rskj-core/src/main/java/co/rsk/pcc/environment/GetCallStackDepth.java

```diff
@@ -21,14 +21,13 @@
 import co.rsk.pcc.NativeMethod;
 import co.rsk.pcc.exception.NativeContractIllegalArgumentException;
 import org.ethereum.core.CallTransaction;
-import org.ethereum.util.ByteUtil;
 import org.ethereum.vm.program.invoke.ProgramInvoke;
 
 public class GetCallStackDepth extends NativeMethod {
     private final CallTransaction.Function function = CallTransaction.Function.fromSignature(
             "getCallStackDepth",
             new String[]{},
-            new String[]{"bytes"}
+            new String[]{"uint"}
```

Comment:
It's a different type in the spec. Check [this](https://github.com/rsksmart/rskj/pull/2190#pullrequestreview-1747883059) comment.

Reply:
I'll fix it right away.